patches/karma-edgium-launcher+4.0.0-0.patch

```diff
@@ -0,0 +1,423 @@
+diff --git a/node_modules/karma-edgium-launcher/index.js b/node_modules/karma-edgium-launcher/index.js
```

Comment:
Could we add a `karma-edgium-launcher.md` file in this folder to specify why we need this patch and when it could be possibly removed?